requirements.txt

```diff
@@ -11,6 +11,8 @@ jmespath
 openstacksdk
 packaging
 python-dateutil
+python-novaclient
```

Comment:
Based on the documentation and module source code, os_server_actions should only require shade.

https://docs.ansible.com/ansible/2.3/os_server_actions_module.html
https://github.com/ansible/ansible/blob/v2.3.2.0-1/lib/ansible/modules/cloud/openstack/os_server_actions.py

Reply:
Without python-novaclient installed, using os_server_actions to lock/unlock throws this:

```
UserWarning: Using shade to get a novaclient object is deprecated. If you need a novaclient object, please use make_legacy_client in os-client-config instead
```